### DISCUSSION TITLE
Table Setup (react-aria checkbox issue)

### DISCUSSION THREAD

**Discussion by golansoffer** [adobe/react-spectrum#5135] [Q&A] [answered]
Hey, everyone! I'm currently using react-aria to build our design system at my company, and I'm working on the `<Table />` component.

I've easily set up the initial skeleton using `react-stately`, but I occasionally encounter minor issues that aren't covered in the documentation. So, if react-aria/react-spectrum has a Discord channel or another community tool for interaction, I'd appreciate hearing about it.

Basically, I've been trying to follow the docs to set up selectable rows with checkbox cells. It works perfectly when I use the native `input[type="checkbox"]`. However, if I use a different checkbox indicator (visually hidden input with a custom visible SVG for example), the checkbox doesn't stay checked; it appears to check momentarily, but when I release the mouse while it's still over the 'checkbox indicator,' it becomes unchecked.

I'm wondering if any of you have experienced this issue before.

**Comment by reidbarber** [comment, marked as answer]
> So, if react-aria/react-spectrum has a Discord channel or another community tool for interaction, I'd appreciate hearing about it.

Not at the moment! Currently Github is the primary means, via Issues or Discussions. 

> However, if I use a different checkbox indicator (visually hidden input with a custom visible SVG for example), the checkbox doesn't stay checked; it appears to check momentarily, but when I release the mouse while it's still over the 'checkbox indicator,' it becomes unchecked.

We use an SVG for the checkboxes in our React Aria Components [Table](https://react-spectrum.adobe.com/react-aria/Table.html) example in the docs, so you could compare your setup to that implementation ([Table source](https://github.com/adobe/react-spectrum/blob/main/packages/react-aria-components/src/Table.tsx) / [Checkbox source](https://github.com/adobe/react-spectrum/blob/main/packages/react-aria-components/src/Checkbox.tsx)). If that doesn't help, feel free to post a codesandbox link reproducing this so we can take a deeper look.

**Reply by snowystinger** [reply]
This might be of use as well https://github.com/adobe/react-spectrum/issues/2383

**Comment by golansoffer** [comment]
Thank you guys
great work btw
